src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -111,76 +83,46 @@ export const ProfilePictureInput: React.FunctionComponent<ProfilePictureInputPro
-    },
-    [setValue]
-  );
-
-  const isLoading =
-    (isSovereignVenueIdLoading || isLoadingCustomAvatars) &&
-    (customAvatars !== undefined || error !== undefined);
-
-  const defaultAvatars = customAvatars?.length
-    ? customAvatars
-    : DEFAULT_AVATARS;
-
-  const avatarImages = useMemo(
-    () =>
-      [githubImageSrc, ...defaultAvatars]
-        .filter(isDefined)
-        .map((avatarSrc, index) => (
-          <div
-            key={`${avatarSrc}-${index}`}
-            className="profile-picture-preview-container"
-            onClick={() => uploadDefaultAvatar(avatarSrc)}
-          >
-            <img
-              src={avatarSrc}
-              className="profile-icon profile-picture-preview"
-              alt={`default avatar ${index}`}
-            />
-          </div>
-        )),
-    [defaultAvatars, githubImageSrc, uploadDefaultAvatar]
-  );
-
   return (
     <div className="profile-picture-upload-form">
       <div
         className="profile-picture-preview-container"
         onClick={() => uploadRef.current?.click()}
       >
         <img
-          src={pictureUrl || "/default-profile-pic.png"}
+          src={pictureUrl || "/avatars/default-octocat-3.png"}
           className="profile-icon profile-picture-preview"
           alt="your profile"
         />
       </div>
-      <input
-        type="file"
-        id="profile-picture-input"
-        name="profilePicture"
-        onChange={handleFileChange}
-        accept={ACCEPTED_IMAGE_TYPES}
-        className="profile-picture-input"
-        ref={uploadRef}
-      />
-      <label htmlFor="profile-picture-input" className="profile-picture-button">
-        Upload your profile pic
-      </label>
+      <p className="profile-picture-options">
+        Want to change your profile photo? Take a{" "}
+        <a
+          className="button--a"
+          href="https://virtual.githubphotobooth.com/virtual/capture/gr99n"
+          {...externalUrlAdditionalProps}
+        >
+          Summit snap
+        </a>{" "}
+        or{" "}
+        <input
+          type="file"
+          id="profile-picture-input"
+          name="profilePicture"
+          onChange={handleFileChange}
+          accept={ACCEPTED_IMAGE_TYPES}
+          className="profile-picture-input"
+          ref={uploadRef}
+        />
+        <label htmlFor="profile-picture-input" className="button--a">
+          upload a photo.
+        </label>
+      </p>
       {errors.pictureUrl && errors.pictureUrl.type === "required" && (
         <span className="input-error">Profile picture is required</span>
       )}
       {isPictureUploading && <small>Picture uploading...</small>}
       {error && <small>Error uploading: {error}</small>}
-      <small>
-        Feeling camera shy? Choose from one of these Summit profile avatars.
-      </small>
-      <div className="default-avatars-container">
-        {isLoading ? <div>Loading...</div> : avatarImages}
-      </div>
```

Comment:
Why was all of this removed?

Reply:
same as above (we had to get rid of the default avatars)